x/oracle/keeper/upgrade.go

```diff
@@ -36,3 +38,32 @@ func (k Keeper) GetOracleUpgradeInfo(ctx sdk.Context) (*types.OracleUpgradeInfo,
 
 	return &upgradeInfo, nil
 }
+
+func (k Keeper) UpgradeOracle(ctx sdk.Context, msg *types.MsgUpgradeOracle) error {
+	oracleRegistration := msg.ToOracleRegistration()
```

Comment:
IMO, the name of `ToOracleRegistration` is little bit confused. How about changing the name to `UpgradeOracleRegistration`?

Reply:
Thank you comment.
This function is changing MsgUpgradeOracle to OracleRegistration.
Umm... I don't think this name would be interpreted differently.

Reply:
Ah, I misunderstood. For now, I think we should keep the name as it is.